.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.2
```

Comment:
Unrelated but I wanted this for local development. `asdf` doesn't work for me

Reply:
I think you can add this to `.git/info/exclude` or do `git update-index --skip-worktree .ruby-version` to keep it out of the repo.